packages/editor/src/components/autocompleters/block.js

```diff
@@ -54,7 +54,7 @@ export function createBlockCompleter( {
 } = {} ) {
 	return {
 		name: 'blocks',
-		className: 'editor-autocompleters__block',
+		className: 'block-editor-autocompleters__block',
```

Comment:
I think this className shouldn't be updated since it's the editor package. Any reasons for this @aduth and are there other similar use-cases?

Reply:
This does appear to have been included in the updates from `editor-` to `block-editor-` in #14420, and in retrospect, I can't see any reason why it should have been (there are no other references outside the `editor` package). I think the correct course of action then would be to change the applied styles back from `.block-editor-` to `.editor-` in `src/components/autocompleters/style.scss`.

https://github.com/WordPress/gutenberg/pull/14420/files#diff-ea8f473f0650959558c8075aac355a73

This appears to have been the only file changed in `packages/editor/` in #14420.